runtime/commonMain/src/kotlinx/serialization/builtins/CollectionSerializers.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.builtins
+
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.*
+
+/**
+ * Creates a serializer for [`List<T>`][List] for the given serializer of type [T].
+ */
+public val <T> KSerializer<T>.list: KSerializer<List<T>>
+    get() = ArrayListSerializer(this)
+
+/**
+ * Creates a serializer for [`List<T>`][List] for the given serializer of type [T].
+ */
+public fun <T> ListSerializer(elementSerializer: KSerializer<T>): KSerializer<List<T>> =
```

Comment:
Is such shorthand needed, when anyone can just use `serializer<List<T>>()` and it will produce the the same result, no?
What is the purpose of it?

Reply:
It can be used with custom serializer, for example, or with polymorphic/contextual one.
Another reason can be `typeOf` avoidance -- one may want to avoid experimental stdlib API or `typeOf` specifically for whatever reason

Reply:
oh, I added comment to wrong line, they are too similar...
I want to add it to next function
`public inline fun <reified T> ListSerializer(): KSerializer<List<T>>`
The same question :)

Reply:
Just a handy and a bit more lightweight alias